builds/linux/rhel/9/linux-rhel.pkr.hcl

```diff
@@ -74,6 +75,8 @@ locals {
         lvm        = var.vm_disk_lvm
       })
       additional_packages = join(" ", var.additional_packages)
+      rpm_gpg_keys = var.rpm_gpg_keys
+      yum_repositories = var.yum_repositories
```

Comment:
Should this be only for RHEL or expanded to the other RHEL-based distributions?

Reply:
yes it is a good idea

Reply:
So this should ideally be added to support the other RHEL-based distributions in the project.